data/mods/Magiclysm/worldgen/forest_tomb.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "type": "palette",
+    "id": "tomb",
+    "terrain": {
+      ".": [ "t_rock_floor" ],
+      " ": [ [ "t_grass_dead", 2 ], [ "t_dirt", 15 ], [ "t_dirtmound", 3 ], [ "t_rock_floor_no_roof", 8 ] ],
+      "T": [ "t_tree_dead" ],
+      "S": [ "t_shrub" ],
+      "<": [ "t_slope_up" ],
+      ">": [ "t_slope_down" ],
+      "#": [ "t_rock" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "forest_tomb" ],
+    "//": "Ground level - Z-0.",
+    "weight": 100,
+    "object": {
+      "rows": [
+        "                  S     ",
+        "            S           ",
+        "   S                    ",
+        "                    S   ",
+        "        S               ",
+        "   S            S       ",
+        "   ##################   ",
+        "   ##################   ",
+        " S ####..##>>##...###   ",
+        "   ###$..........####S  ",
+        "   ###...##..##...###   ",
+        "   ###.####..########   ",
+        "   ###..###..$..#####   ",
+        "   ###...##..#...x### S ",
+        "  S####x###..########   ",
+        "   ########..########   ",
+        "          T  T     S    ",
+        "     S                  ",
+        "          T  T       S  ",
+        "      S                 ",
+        "          T  T          ",
+        "                 S      ",
+        "    S                   "
+      ],
+      "palettes": [ "tomb" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "forest_tomb_bottom" ],
+    "//": "Underground level - Z -1.",
+    "weight": 100,
+    "object": {
+      "rows": [
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "########################",
+        "######.x################",
+        "######.####.......######",
+        "######.####..###.$######",
+        "#####....##<<##x..######",
+        "#######.###..###########",
+        "#######.....############",
+        "###########..###########",
+        "###########..###########",
+        "############.###########",
+        "###########..###########",
+        "###########..###########",
+        "###########.############",
+        "########........########",
+        "########........########",
+        "########...HZ...########",
```

Comment:
Should H and Z have a definition in the palette? I couldn't find them. I'm excited to see this.  Forests feel so empty sometimes.

Reply:
H is going to be altar.
Z is for boss/main monster.